server/services/reportedAdjudicationsService.ts

```diff
@@ -309,7 +313,13 @@ export default class ReportedAdjudicationsService {
         reportedAdjudication.incidentDetails.dateTimeOfIncident,
         'D MMMM YYYY - HH:mm'
       ),
+      dateTimeOfDiscovery: reportedAdjudication.incidentDetails.dateTimeOfDiscovery,
+      formattedDateTimeOfDiscovery: formatTimestampToDate(
+        reportedAdjudication.incidentDetails.dateTimeOfDiscovery,
+        'D MMMM YYYY - HH:mm'
+      ),
       statusDisplayName: reportedAdjudicationStatusDisplayName(reportedAdjudication.status),
+      formattedDateTimeOfScheduledHearing: formatTimestampToDate(nextSheduledHearingDate, 'D MMMM YYYY - HH:mm'),
```

Comment:
You should be able to use 
`formatTimestampToDate(nextSheduledHearingDate, 'D MMMM YYYY - HH:mm') || "-",` here instead to show the hyphen when there's no hearing to display

Reply:
I think this means you'll have to correct a few of the tests in the `reportedAdjudicationsService.test.ts` by the way, as it looks like in some of the expected data it returns an empty string but now it will returns "-"